SpatialGDK/Source/SpatialGDKFunctionalTests/Private/Test2x2GridStrategy.cpp

```diff
@@ -0,0 +1,10 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#include "Test2x2GridStrategy.h"
+
+UTest2x2GridStrategy::UTest2x2GridStrategy()
+{
+	Rows = 2;
+	Cols = 2;
+	InterestBorder = 10000.0f;
```

Comment:
Any specific reason for 10000?

Reply:
just a lot

Reply:
could go for even more

Reply:
could add a comment?

Reply:
Changed to max of worldWidth and worldHeight